### ISSUE TITLE
Ability to reuse custom tags

### ISSUE THREAD

**Issue by timhot** [mcuz/cacophony-audiotagger#2]
When I enter a custom tag, it would be good to have that available to use there after.

**Comment by marknikora** [comment]
easiest way to implement is user specific (tags not shareable between users), last for current session (browser window) only
possible to retain these in local storage on browser

**Comment by timhot** [comment]
To start with anyone can add new tags that are added to a table of tags on the server.
Later, only admin has the ability to add new tags to the table.

**Comment by Stormrose** [comment]
> When I enter a custom tag, it would be good to have that available to use there after.
Agreed it's probably better, at least in the beginning, to limit tag reuse to individual users. Implementation-wise, local browser storage is fine for now and can be reviewed later.

> Later, only admin has the ability to add new tags to the table.
Let's see how this goes. In my experience with folksonomies, an underlying ontology does eventually emerge and then you can build rules around it. It might be better to enforce restrictions at the UI level since the UIs are probably tuned for the types of data they need to collect. An exception to this would be if the APIs because much more open and there were many many third-party UIs contributing data.
